### DISCUSSION TITLE
any view to report all in pdf or excel? export button?

### DISCUSSION THREAD

**Discussion by mathcenas** [kromitgmbh/titra#63] [Q&A] [answered]
we have our titra in docker env, is there a way to export all time from a project or client, in pdf or xls?. thankS!

**Comment by faburem** [comment, marked as answer]
Yes, that is possible in the Details view using the Excel button:
![image](https://user-images.githubusercontent.com/11456790/128321260-07a8f1ca-8ef2-4bab-bf19-88004c8154ed.png)
